canonical/store-components
# Fix card footers

## Done
Fixed card footers to the bottom of the card component

## QA
- Go to https://store-components-63.demos.haus/?path=/story/charmcard--default-story
- Check that the footer is at the bottom of the card component

## Issue
Fixes #62
